package CallExceptionMethods;

public class Main {

    public static void main(String[] args) {
        try {
            // Call a method here that may throw an exception
        } catch (Exception e) {
            // Handle the exception here
        }
    }

}
